src/components/Preferences/ResourceValidation/ResourceValidationSettings.tsx

```diff
@@ -0,0 +1,187 @@
+import { useTranslation } from 'react-i18next';
+import { useRecoilState, useRecoilValue } from 'recoil';
+import { Switch } from 'fundamental-react';
+import {
+  getExtendedValidateResourceState,
+  validateResourcesState,
+} from 'state/preferences/validateResourcesAtom';
+import { validationSchemasState } from 'state/validationSchemasAtom';
+import { useMemo } from 'react';
+import { GenericList } from 'shared/components/GenericList/GenericList';
+
+import './ResourceValidationSettings.scss';
+import { useFeature } from 'hooks/useFeature';
+import { ValidationFeatureConfig } from 'state/validationEnabledSchemasAtom';
+
+export default function ResourceValidationSettings() {
+  const { t } = useTranslation();
+  const [validateResources, setValidateResources] = useRecoilState(
+    validateResourcesState,
+  );
+  const validationFeature = useFeature(
+    'RESOURCE_VALIDATION',
+  ) as ValidationFeatureConfig;
+
+  const {
+    enabled,
```

Comment:
As I understood this is geting enabled from policy, so we can pare enabled to isEnabled
```suggestion
    enabled: isEnabled,
```